stock_no_negative/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 
 {
     'name': 'Stock Disallow Negative',
-    'version': '12.0.1.0.0',
+    'version': '12.0.1.0.1',
```

Comment:
Don't change the module version. Now this is done by the bot.

Reply:
@pedrobaeza Ok!